doc/how_to/streamlit_migration/chat.md

```diff
@@ -4,15 +4,15 @@ Both Streamlit and Panel provides special components to help you build conversat
 
 | Streamlit            | Panel               | Description                            |
 | -------------------- | ------------------- | -------------------------------------- |
-| [`chat_message`](https://docs.streamlit.io/library/api-reference/chat/st.chat_message)    | [`ChatMessage`](../../../examples/reference/chat/ChatMessage.ipynb) | Display a chat message  |
+| [`chat_message`](https://docs.streamlit.io/library/api-reference/chat/st.chat_message)    | [`ChatMessage`](../../reference/chat/ChatMessage.md) | Display a chat message  |
```

Comment:
Are the reference guides turned into Markdown files at some point during the docs build process?

Reply:
I don't know. But I guess so. And it seems to work.

Reply:
I don't think they are but if it works, it works. I would suggest the correct fix is to remove the file extension entirely though (if that works).